src/commands/lambda/flare-command-validator.ts

```diff
@@ -0,0 +1,89 @@
+// Uses the Chain of Responsibility design pattern to validate the command
+// eslint-disable-next-line max-classes-per-file
+abstract class RequestHandler {
+  private nextHandler: RequestHandler | undefined
+
+  public setNext(handler: RequestHandler): RequestHandler {
+    this.nextHandler = handler
+
+    return handler
+  }
+
+  public handle(request: {[key: string]: any}): string | undefined {
```

Comment:
What does `handle` mean? I don't understand the name since what we are doing is checking if something is valid.

Reply:
The method name of `handle` is based on the Chain of Responsibility pattern
https://refactoring.guru/design-patterns/chain-of-responsibility
Do you think I should rename it to something like `validate`?

Reply:
I understand. But this classes are meant to be used as a blueprint in which you then transform for the use case. In this case, `handle` should be something meaningful to what we are doing here.

Reply:
I see, that makes sense. Renamed